pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -91,21 +111,23 @@ func populateInitialVolumeStatus(ctx *volumemgrContext, dirName string) {
 
 	for _, location := range locations {
 		filelocation := dirName + "/" + location.Name()
-		if location.IsDir() {
+		if location.IsDir() && !isContainer {
 			log.Debugf("populateInitialVolumeStatus: directory %s ignored", filelocation)
 			continue
 		}
-
+		var size int64
 		info, err := os.Stat(filelocation)
 		if err != nil {
 			log.Errorf("Error in getting file information. Err: %s. "+
 				"Deleting file %s", err, filelocation)
 			deleteFile(filelocation)
 			continue
 		}
-
-		size := info.Size()
-		_, sha256, appUUIDStr, purgeCounter := parseAppRwVolumeName(filelocation)
+		size = info.Size()
+		if isContainer {
+			size, _ = dirSize(filelocation)
+		}
```

Comment:
Cleaner to check isContainer and not do the os.Stat in that case,

Reply:
Actually, we need to do os.Stat on the directory also because we need last modified time which we are filling in VolumeStatus here.
https://github.com/lf-edge/eve/blob/fc1443a69e2b1d11d9c2aecc93acbc0693581cb1/pkg/pillar/cmd/volumemgr/initialvolumestatus.go#L156

Reply:
OK